apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import styleConstants from '@cdo/apps/styleConstants';
+import HeaderBanner from '../HeaderBanner';
+import CourseCatalogIllustration01 from '../../../static/curriculum_catalog/course-catalog-illustration-01.png';
+
+class CurriculumCatalog extends Component {
+  static propTypes = {
+    curriculaData: PropTypes.arrayOf(
+      PropTypes.shape({
```

Comment:
in the future you can consider extracting this shape into a shapes.jsx file, since it is likely to be reused in other components. cc @megcrenshaw